modules/interacting-serverless-apps-http2-gRPC-up-to-4-9.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// serverless/knative-serving/external-ingress-routing/using-http2-gRPC.adoc
+
+:_content-type: PROCEDURE
+[id="interacting-serverless-apps-http2-gRPC-up-to-4-9_{context}"]
```

Comment:
IDs should not have capital letters in them (my understanding is that they can break Pantheon builds) . Can you make these lowercase?